crates/bevy_render/Cargo.toml

```diff
@@ -29,7 +29,7 @@ zstd = ["ruzstd"]
 
 trace = ["profiling"]
 tracing-tracy = []
-wgpu_trace = ["wgpu/trace"]
+wgpu_trace = []
```

Comment:
wgpu_trace was temporarily removed https://github.com/gfx-rs/wgpu/issues/5974. We should remove our feature for now too imo (and the corresponding docs we have referencing it).

Reply:
Hi, I was browsing around and looking at Bevy's PRs, so please excuse me if I seem a bit out of place.

I assume that if the `wgpu_trace` feature is removed, it'll be re-added back eventually? If so, then I might recommend keeping the `wgpu_trace` feature, but have a note somewhere (in the docs? during compilation? as a compile error? etc.) that the feature does not currently work, linking to the wgpu issue you mentioned.

That said, if you actually mean to remove it permanently, then my suggestion would be nil.

(P.S. Personally, I don't see the use in it, compared to just saying "pull in `wgpu` and include the `trace` feature". But that's me - maybe bevy has the `wgpu_trace` feature for a reason I'm not aware of?)

Reply:
I'm in favor of just removing it outright, but if we really wanted to keep it, putting in a compile error with a link to an issue seems like a good idea to me.